validator/validate_api.py

```diff
@@ -277,7 +283,7 @@ def validate_response(
 # credentials are needed so the SSO cookie can be read
 @bp.route("/validate", methods=("GET", "POST"))
 @cross_origin(supports_credentials=True)
-def validation_api_entry():
+def validation_api_entry(feature_weights_set_id):
```

Comment:
remove feature_weights_set_id as a param here: it weill never be set by the way this is called by flask

Reply:
Params like that are for pairing with parts of the url: e.g. `/validate/<feature_weights_set_id>` would then require the set_id uuid as a url component.